### ISSUE TITLE
Error: Can't Load URL.

### ISSUE THREAD

**Issue by jeff93** [stefanbund/311#3]
Hi Professor Bund! I have edited the settings it's describing by adding the domain to the App Settings, but i am still getting the error. I'm stuck here and not sure where to go next. Any help is appreciated!

![screen shot 2016-10-04 at 8 50 55 pm](https://cloud.githubusercontent.com/assets/18152046/19100504/4a18575a-8a74-11e6-8b7b-70ee3220a834.png)
![screen shot 2016-10-04 at 8 50 20 pm](https://cloud.githubusercontent.com/assets/18152046/19100506/4c5cd180-8a74-11e6-8ebe-0b51c5340e27.png)

Here is the URL to my bucket and sample.HTML: https://s3-us-west-2.amazonaws.com/cisstuff/sample.HTML

**Comment by jeff93** [comment]
Okay so I sort of fixed it. I am still getting an error in one tab, however, another tab opens up before letting me choose a file but no upload button appears. I checked my code and the upload handling code is present. I'm not sure what I might be missing :/
![screen shot 2016-10-04 at 8 53 58 pm](https://cloud.githubusercontent.com/assets/18152046/19100592/0112c22e-8a75-11e6-96dd-08d298c77145.png)

**Comment by stefanbund** [comment]
Open up the console, using inspect element

From Stefan

**Comment by jeff93** [comment]
Figured it out works perfectly now thanks Professor!

**Comment by stefanbund** [comment]
Glad to hear of your progress. Could you summarize your solution in one screenshot? This might help others with the same issue

From Stefan
